test/e2e-test/pages/ask-for-more-time/ask-for-more-time.ts

```diff
@@ -0,0 +1,21 @@
+import { paths } from '../../../../app/paths';
+
+const config = require('config');
+
+const testUrl = config.get('testUrl');
+
+module.exports = {
+  askForMoreTime(I) {
+    Then(/^I click Ask for more time$/, async () => {
+      await I.amOnPage(testUrl + paths.askForMoreTime);
```

Comment:
This could lead to confusion if you say you click but then inside you are navigating to that page...

Change the title of the function maybe and say `I navigate to Ask for more time page`...